common/vpcclient/vpcfilevolume/file_share_service.go

```diff
@@ -30,6 +30,10 @@ import (
 //
 //go:generate counterfeiter -o fakes/share.go --fake-name FileShareService . FileShareManager
 type FileShareManager interface {
+
+	// Get the share profile by using profile name
+	GetShareProfile(profileName string, ctxLogger *zap.Logger) error
```

Comment:
Method name is not aligned with implementation, can you implement actual GetShareProfile which will return profile details with error and you can use as per your requirements